src/algebra/order/field_defs.lean

```diff
@@ -0,0 +1,45 @@
+/-
+Copyright (c) 2014 Robert Lewis. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Robert Lewis, Leonardo de Moura, Mario Carneiro, Floris van Doorn
+-/
+import algebra.field.basic
+import algebra.group_power.lemmas
+import algebra.group_power.order
+import algebra.order.ring
+import order.bounds
+import tactic.monotonicity.basic
```

Comment:
You should review the imports, I don't think you need all of these.

Reply:
probably not but I'd rather just do it this way as it's a quick hack whilst we wait for a lean bugfix